src/components/implementation/no_interface/boot_test/test_booter.c

```diff
@@ -0,0 +1,43 @@
+#include <stdio.h>
+#include <string.h>
+#include <cos_kernel_api.h>
+#include "boot_deps.h"
+
+static void
+cos_llprint(char *s, int len)
+{ call_cap(PRINT_CAP_TEMP, (int)s, len, 0, 0); }
+
+int
+prints(char *s)
```

Comment:
See previous comments on removing this into `llprint.h` in `src/components/include/`